SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -150,7 +150,7 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 	// We use this to indicate if a new Actor should be created or to link a pre-existing Actor
 	// when receiving an AddEntityOp.
 	TSchemaOption<FUnrealObjectRef> StablyNamedObjectRef;
-	if (Actor->IsFullNameStableForNetworking())
+	if (Actor->HasAnyFlags(RF_WasLoaded))
```

Comment:
Why was this changed? Are the other conditions in `IsFullNameStableForNetworking` not relevant?

Reply:
`IsFullNameStableForNetworking` relies on `bNetStartup`, which is only going to be set if `bNetLoadOnClient` is true, while we're interested in all the loaded actors.
@Vatyx should probably add a comment.

Reply:
Can do.